Assembly plugin non-fatal errors on Windows

**Describe the bug**
Project builds (with unit tests disabled), but there are several errors on Windows

**To Reproduce**
run "mvn package -Dmaven.test.skip=true"
See error:
`[ERROR] OS=Windows and the assembly descriptor contains a *nix-specific root-relative-reference (starting with slash) /
`
**Expected behavior**
There should be no errors

** Version of Software Used**
master branch
Windows 10

--- a/src/main/assembly/tar-assembly.xml
+++ b/src/main/assembly/tar-assembly.xml
@@ -95,7 +95,7 @@ POSSIBILITY OF SUCH DAMAGE.
   <files>
     <file>
       <source>LICENSE.txt</source>
-      <outputDirectory>/</outputDirectory>
+      <outputDirectory/>
       <fileMode>644</fileMode>
       <lineEnding>unix</lineEnding>
     </file>

--- a/src/main/assembly/zip-assembly.xml
+++ b/src/main/assembly/zip-assembly.xml
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="UTF-8"?>
 
 <!--
 Copyright © 2019, California Institute of Technology ("Caltech").
@@ -95,7 +95,7 @@ POSSIBILITY OF SUCH DAMAGE.
   <files>
     <file>
       <source>LICENSE.txt</source>
-      <outputDirectory>/</outputDirectory>
+      <outputDirectory/>
       <fileMode>644</fileMode>
       <lineEnding>unix</lineEnding>
     </file>

